cli/pom.xml

```diff
@@ -137,6 +137,24 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
```

Comment:
`maven-localizer-plugin:generate` already adds this source to the source list - so this has the potential to add the same directory twice (not sure how Maven would actually handle that - but I am expecting it is not the best)